unstructured/ingest/cli/cmds/google_drive.py

```diff
@@ -1,20 +1,18 @@
 import typing as t
 from dataclasses import dataclass
+from pathlib import Path
 
 import click
 
 from unstructured.ingest.cli.base.src import BaseSrcCmd
-from unstructured.ingest.cli.interfaces import (
-    CliMixin,
-    CliRecursiveConfig,
-)
+from unstructured.ingest.cli.interfaces import CliMixin, CliRecursiveConfig, FileOrJson
 from unstructured.ingest.interfaces import BaseConfig
 
 
 @dataclass
 class GoogleDriveCliConfig(BaseConfig, CliMixin):
     drive_id: str
-    service_account_key: str
+    token: t.Union[dict, Path]
```

Comment:
I think in either case we should keep the old key and flag as deprecated.
let's defer to service_account_key as the one we make common to both, since this is more specific and seems to be more wildly referenced in their documents.

Reply:
What do we use if both are provided? Raise an error?

Reply:
actually, I think google drive is the one folks are really using. so I think fine to not deprecate, but let's just make service_account_key the common one.